[win10] run coverage -> rm is not defined

Hi!
I try `npm run coverage`, but got Exception.
Tell me, please, where does `rm()` function come from?

**Tell us about your environment:**
* Puppeteer version: 1.2.0-post
* Platform / OS version: MINGW64_NT-10.0
* URLs (if applicable):
* Node.js version: v8.10.0

**What steps will reproduce the problem?**
```
vgorbachev@vgorbachev-l MINGW64 ~/project/fork/puppeteer (master)
$ npm run coverage

> puppeteer@1.2.0-post coverage C:\Users\vgorbachev\project\fork\puppeteer
> cross-env COVERAGE=true npm run unit


> puppeteer@1.2.0-post unit C:\Users\vgorbachev\project\fork\puppeteer
> node test/test.js

C:\Users\vgorbachev\project\fork\puppeteer\test\test.js:68
  rm(OUTPUT_DIR);
  ^

ReferenceError: rm is not defined
    at Object.<anonymous> (C:\Users\vgorbachev\project\fork\puppeteer\test\test.js:68:3)
    at Module._compile (module.js:652:30)
    at Object.Module._extensions..js (module.js:663:10)
    at Module.load (module.js:565:32)
    at tryModuleLoad (module.js:505:12)
    at Function.Module._load (module.js:497:3)
    at Function.Module.runMain (module.js:693:10)
    at startup (bootstrap_node.js:188:16)
    at bootstrap_node.js:609:3
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! puppeteer@1.2.0-post unit: `node test/test.js`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the puppeteer@1.2.0-post unit script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\vgorbachev\AppData\Roaming\npm-cache\_logs\2018-03-20T14_12_26_552Z-debug.log
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! puppeteer@1.2.0-post coverage: `cross-env COVERAGE=true npm run unit`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the puppeteer@1.2.0-post coverage script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\vgorbachev\AppData\Roaming\npm-cache\_logs\2018-03-20T14_12_26_619Z-debug.log
```

--- a/test/test.js
+++ b/test/test.js
@@ -14,6 +14,7 @@
  * limitations under the License.
  */
 const fs = require('fs');
+const rm = require('rimraf').sync;
 const path = require('path');
 const SimpleServer = require('./server/SimpleServer');
 const GoldenUtils = require('./golden-utils');

